src/NameApp.java

```diff
@@ -0,0 +1,313 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.PrintWriter;
+import java.util.Scanner;
+
+/**
+ * Created by trevor.campbell on 4/29/16.
+ */
+public class NameApp {
+
+    public static void main(String[] args) {
+        String[] fileData = new String[4429];
+        Name[] nameList;
+        //load the file
+        Scanner inputStream = null;
+        String fileName = "names.txt";
+
+        try {
+            inputStream = new Scanner (new File(fileName));
+        }
+        catch (FileNotFoundException e){
+            System.out.println("Error opening file named: " + fileName);
+        }
+
+        for (int i = 0; i < fileData.length; i++) {
+            fileData[i] = inputStream.nextLine();
+        }
+
+        nameList = readNamesFile(fileData);
+
+        String choice;
+        boolean stillWorking = true;
+
+        do{
```

Comment:
This is picky, but stylistically, you should probably have a space after this `do`, and before the following `while`. Both because that's how you do it in the other file, and because I think that's generally accepted.